corehq/apps/formplayer_api/smsforms/tests/test_formplayer_interface.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+import requests_mock
+from django.conf import settings
+from django.test import TestCase
+
+from corehq.apps.formplayer_api.smsforms.api import FormplayerInterface
+from corehq.apps.formplayer_api.utils import get_formplayer_url
+from corehq.util.hmac_request import get_hmac_digest
+
+
+class TestFormplayerInterface(TestCase):
```

Comment:
From a brief glance, I don't see anything in these tests accessing the database -- could this be converted to `SimpleTestCase`?

Reply:
Yes they could. I had initially been using a DB object but then dropped it.